tests/mesh/test-point-set.cpp

```diff
@@ -84,6 +84,18 @@ void test_io( const geode::PointSet3D& point_set, const std::string& filename )
     load_point_set( *new_point_set, filename );
 }
 
+void test_copy( const geode::PointSet3D& point_set )
+{
+    auto point_set2 = point_set.clone();
+    OPENGEODE_EXCEPTION( point_set2->nb_vertices() == 3,
+        "[Test] PointSet2 should have 3 vertices" );
+
+    auto attribute =
+        point_set2->vertex_attribute_manager().find_attribute< bool >( "test" );
+    OPENGEODE_EXCEPTION( attribute->value( 0 ) == true,
+        "[Test] PointSet2 attribute value should be true" );
```

Comment:
I suggest to add a test on coordinates values, using this answer:
geode::Point3D answer{ { 2.1, 9.4, 6.7 } };